app/assets/javascripts/modules/search-highlight.es6

```diff
@@ -0,0 +1,29 @@
+{
+  'use strict';
+
+  class SearchHighlight {
+    start(el) {
+      const term = this.getUrlParameter(encodeURIComponent('search[q]')),
+      regEx = new RegExp(`(${term})`, 'gi');
+
+      if (!term) {
+        return;
+      }
+
+      $(el).find('.js-allow-highlighting').each((index, td) => {
+        $(td).html($(td).html().replace(regEx, '<mark>$1</mark>'));
+      });
+    }
+
+    getUrlParameter(name) {
+      name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
```

Comment:
Surprised this passes eslint actually. It's generally a bad idea to manipulate the arguments directly, can we assign this to a new variable?

Tempted to enable this too: http://eslint.org/docs/rules/no-param-reassign